ios/Framework/Core/Portlet/html/index.html

```diff
@@ -8,14 +8,14 @@
 			form.setAttribute('method', method);
 			form.setAttribute('action', '/c/portal/login?redirect=<portletUrl>');
 
-			let params = {
+			var params = {
```

Comment:
Why var now?

Reply:
in iOS 9 we can't use the latest features :(